webpack.config.js

```diff
@@ -124,11 +138,11 @@ module.exports = (env, argv) => {
                 // overflows (https://github.com/webpack/webpack/issues/1721), and
                 // there is no need for webpack to parse them - they can just be
                 // included as-is.
-                /highlight\.js[\\\/]lib[\\\/]languages/,
+                /highlight\.js[\\/]lib[\\/]languages/,
 
                 // olm takes ages for webpack to process, and it's already heavily
                 // optimised, so there is little to gain by us uglifying it.
-                /olm[\\\/](javascript[\\\/])?olm\.js$/,
+                /olm[\\/](javascript[\\/])?olm\.js$/,
```

Comment:
why the windows removal?

Reply:
The linter was complaining about those, was following the reports of ESLint. More than happy to revert

Reply:
I'm a bit surprised our linter cares this much - what was the error?

Reply:
Yup pretty much all errors, there's still a couple warning but VSCode is not shouting at me anymore

Reply:
I'm not getting the same warnings, and our CI doesn't seem to be complaining either. Is this a local issue?

Reply:
It doesn't appear to be local. If I delete `.eslintrc` at the root of `element-web` all the "errors" reported go away, and as soon as I restore the file they reappear

![Screen Shot 2021-04-15 at 17 12 15](https://user-images.githubusercontent.com/769871/114902661-f2311b00-9e0d-11eb-8263-a431c9231d3d.png)

Reply:
that would be our rules file you're deleting, which then causes the defaults to take place. We override the lint rules for things like this.